.rubocop.yml

```
@@ -0,0 +1,120 @@
+---
+AllCops:
+  DisabledByDefault: true
+  Exclude:
+    - bin/**/*
+    - node_modules/**/*
+    - log/**/*
+    - tmp/**/*
+    - vendor/**/*
+    - db/migrate/*
+    - db/schema.rb
+Layout/AccessModifierIndentation:
+  EnforcedStyle: indent
+Layout/BlockAlignment:
+  EnforcedStyleAlignWith: either
+Layout/DefEndAlignment:
+  EnforcedStyleAlignWith: start_of_line
+Layout/DotPosition:
+  EnforcedStyle: leading
+Layout/EmptyComment:
+  Enabled: true
+Layout/EmptyLineBetweenDefs:
+  Enabled: true
+  NumberOfEmptyLines: 1
+Layout/EmptyLinesAroundBlockBody:
+  EnforcedStyle: no_empty_lines
+Layout/EmptyLinesAroundClassBody:
+  EnforcedStyle: no_empty_lines
+Layout/EmptyLinesAroundModuleBody:
+  EnforcedStyle: no_empty_lines
+Layout/EndAlignment:
+  EnforcedStyleAlignWith: start_of_line
+Layout/ExtraSpacing:
+  AllowForAlignment: true
+Layout/IndentArray:
```

This was renamed recently to: 
```suggestion
Layout/IndentFirstArrayElement
```